packages/pilot/src/components/CaptureDetails/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { mapObjIndexed } from 'ramda'
+import {
+  Col,
+  Row,
+} from 'former-kit'
+
+import Property from '../Property'
+
+const renderProperty = (title, value) => (
+  <Property
+    title={title}
+    value={value}
+  />
+)
+
+const fields = (labels, contents) => mapObjIndexed((label, key) =>
+  renderProperty(label, contents[key]),
+labels)
+
+const CaptureDetails = ({
+  contents, labels,
+}) => {
+  const {
+    cardBrand,
+    cardNumber,
+    customerEmail,
+    customerName,
+    installments,
+  } = fields(labels, contents)
+
+  return (
+    <Fragment>
+      { (customerName || customerEmail) &&
```

Comment:
aqui precisa validar com `contents.<prop>` senão ele cria os fields vazios
```suggestion
       { (contents.customerName || contents.customerEmail) &&
```

Reply:
eita bem observado! corrigi em ambos os dados de `customer` e `card`